modules/apps/site/site-admin-web/src/main/resources/META-INF/resources/js/SiteLanguageConfiguration.js

```diff
@@ -121,25 +121,25 @@ export default function SiteLanguageConfiguration({
 
 			{inheritLocales === 'true' ? (
 				<div>
-					<h5 className="h4">
+					<div className="h4">
 						{Liferay.Language.get('default-language')}
-					</h5>
+					</div>
 
 					<p>{defaultLanguageLabel}</p>
 
-					<h5 className="h4">
+					<div className="h4">
 						{Liferay.Language.get('available-languages')}
-					</h5>
+					</div>
 
 					{initialCurrentLanguages
 						.map((language) => language.label)
 						.join(', ')}
 				</div>
 			) : (
 				<fieldset>
-					<h5 className="h4">
+					<div className="h4">
```

Comment:
Where there any tests for these ones that need to be updated to look for .h4 instead of h5?